### DISCUSSION TITLE
Rock 3

### DISCUSSION THREAD

**Discussion by woiza** [Joshua-Riek/ubuntu-rockchip#760] [Ideas]
Hi,

what would it take to build an image for Rock 3 boards?
Orange Pi 3B and Radxa Zero 3, which use pretty much the same SoC, are already supported. Can they be used as a basis?

**Comment by hubby4** [comment]
Support for the Orange Pi 3b, the Radxa Zero 3, and the 3566 SOC is only for Ubuntu 22.04. The Ubuntu 24.04 build removed 3566 support. You'll have to build using the older 22.04 build as a basis.

Read this topic to learn more about why 3566 support has been removed from 24.04. https://github.com/Joshua-Riek/ubuntu-rockchip/issues/723

**Reply by woiza** [reply]
Understood, thank you!

**Comment by ricag101** [comment]
Has anyone had any luck with a Build for the Radxa Rock 3C?
I have attempted to use the Radxa Zero 3 image as a basis, and that ended horribly with the CPU overheating almost instantly.
Any guidance on how to either Build a new Build for the Rock 3C or guidance on how to resolve the use of the Zero 3 as a basis, would be great.

Thanks
